api/src/main/java/org/ehcache/spi/loaderwriter/CacheLoaderWriterConfiguration.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.spi.loaderwriter;
+
+import org.ehcache.spi.service.ServiceConfiguration;
+
+/**
+ * @author Abhilash
+ *
+ */
+public interface CacheLoaderWriterConfiguration extends ServiceConfiguration<CacheLoaderWriterFactory> {
```

Comment:
And the same for the `extends ServiceConfiguration<CacheLoaderWriterFactory>` - why is it needed to intersect at this level with the `CacheLoaderWriter` configuration system?